app/src/androidTest/java/com/sdp13epfl2021/projmag/ProjectEditTest.kt

```diff
@@ -0,0 +1,42 @@
+package com.sdp13epfl2021.projmag
+
+import android.content.Context
+import android.content.Intent
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.*
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.sdp13epfl2021.projmag.model.ImmutableProject
```

Comment:
An import was missing.
```suggestion
import com.sdp13epfl2021.projmag.activities.ProjectCreationActivity
import com.sdp13epfl2021.projmag.model.ImmutableProject
```